src/ui/widgets/StandingsWidget/SessionClock/SessionClock.module.scss

```
@@ -1,7 +1,38 @@
+}
+
+.clockWarning {
+  color: $widget-status-warning;
+}
+
+.clockCritical {
+  color: $widget-status-danger;
+  animation: sessionClockPulse 2s cubic-bezier(0.4, 0, 0.6, 1) infinite;
+}
+
+@keyframes sessionClockPulse {
```

Stylelint (17.14.0)

[error] Expected keyframe name "sessionClockPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)